cypress/test/search-facet.spec.ts

```diff
@@ -1,53 +1,104 @@
-      SearchFacetSteps.getSelectedFacetList().should('have.length', 1);
-      SearchFacetSteps.getSelectedFacet(0).should('contain', 'Recruiting');
+      SearchCheckboxFacetSteps.getSelectedFacetList().should('have.length', 1);
+      SearchCheckboxFacetSteps.getSelectedFacet(0).should('contain', 'Recruiting');
 
-      SearchFacetSteps.clickFacet(1);
-      SearchFacetSteps.getSelectedFacetList().should('have.length', 2);
-      SearchFacetSteps.getSelectedFacet(1).should('contain', 'Temporarily not available');
+      SearchCheckboxFacetSteps.clickFacet(1);
+      SearchCheckboxFacetSteps.getSelectedFacetList().should('have.length', 2);
+      SearchCheckboxFacetSteps.getSelectedFacet(1).should('contain', 'Temporarily not available');
 
-      SearchFacetSteps.clickFacet(2);
-      SearchFacetSteps.getSelectedFacetList().should('have.length', 3);
-      SearchFacetSteps.getSelectedFacet(2).should('contain', 'Enrolling by invitation');
+      SearchCheckboxFacetSteps.clickFacet(2);
+      SearchCheckboxFacetSteps.getSelectedFacetList().should('have.length', 3);
+      SearchCheckboxFacetSteps.getSelectedFacet(2).should('contain', 'Enrolling by invitation');
 
-      SearchFacetSteps.clickFacet(2);
-      SearchFacetSteps.getSelectedFacetList().should('have.length', 2);
-      SearchFacetSteps.getSelectedFacet(2).should('not.be.visible');
+      SearchCheckboxFacetSteps.clickFacet(2);
+      SearchCheckboxFacetSteps.getSelectedFacetList().should('have.length', 2);
+      SearchCheckboxFacetSteps.getSelectedFacet(2).should('not.be.visible');
 
-      SearchFacetSteps.clickFacet(1);
-      SearchFacetSteps.clickFacet(0);
-      SearchFacetSteps.getSelectedFacetList().should('have.length', 0);
+      SearchCheckboxFacetSteps.clickFacet(1);
+      SearchCheckboxFacetSteps.clickFacet(0);
+      SearchCheckboxFacetSteps.getSelectedFacetList().should('have.length', 0);
     });
 
     it('Should deselect all facets outside component', () => {
-      SearchFacetSteps.visit();
-      SearchFacetSteps.clickFacet(1);
-      SearchFacetSteps.clickFacet(2);
-      SearchFacetSteps.clickFacet(3);
-      SearchFacetSteps.clickFacet(4);
-      SearchFacetSteps.clickFacet(5);
+      SearchCheckboxFacetSteps.visit();
+      SearchCheckboxFacetSteps.clickFacet(1);
+      SearchCheckboxFacetSteps.clickFacet(2);
+      SearchCheckboxFacetSteps.clickFacet(3);
+      SearchCheckboxFacetSteps.clickFacet(4);
+      SearchCheckboxFacetSteps.clickFacet(5);
 
-      SearchFacetSteps.getSelectedFacetList().should('have.length', 6);
-      SearchFacetSteps.getDeselectButton().click();
-      SearchFacetSteps.getSelectedFacetList().should('have.length', 0);
+      SearchCheckboxFacetSteps.getSelectedFacetList().should('have.length', 6);
+      SearchCheckboxFacetSteps.getDeselectButton().click();
+      SearchCheckboxFacetSteps.getSelectedFacetList().should('have.length', 0);
     });
-  })
+  });
 
+  context('Date range facet', () => {
```

Comment:
When more tests are added, this spec will become big. As long as each facet can be tested separately, it's better to have separate specs for each facet.